### PR TITLE
Update graph doc for additional scopes

### PR DESCRIPTION
**Pull request by jahsu-MSFT**

### CODE REVIEW

**Review by braedenp-msft — APPROVED** [review]

File: api-reference/beta/api/print-get-settings.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To use the Universal Print service, the user or app's tenant must have an active
 
 |Permission type | Permissions (from least to most privileged) |
 |:---------------|:--------------------------------------------|
-|Delegated (work or school account)| User.Read |
+|Delegated (work or school account)| PrintSettings.Read.All |
```
Comment (nilakhan) [inline api-reference/beta/api/print-get-settings.md:25]: also, PrintSettings.ReadWrite.All

File: api-reference/beta/api/printdocument-createuploadsession.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ In addition to the following permissions, the user or app's tenant must have an
 
 | Permission type                        | Permissions (from least to most privileged) |
 |:---------------------------------------|:--------------------------------------------|
-| Delegated (work or school account)     | PrintJob.Create, PrintJob.ReadWrite, PrintJob.ReadWrite.All |
```
Comment (nilakhan) [inline api-reference/beta/api/printdocument-createuploadsession.md:27]: I think we should preserve the original order as PrintJob.Create is the least privileged scope and PrintJob.ReadWrite.All is the most privileged scope amongst these 3

File: api-reference/beta/api/printer-post-jobs.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ In addition to the following permissions, the user or app's tenant must have an
 
 |Permission type | Permissions (from least to most privileged) |
 |:---------------|:--------------------------------------------|
-|Delegated (work or school account)| PrintJob.Create, PrintJob.ReadWriteBasic, PrintJob.ReadWrite, PrintJob.ReadWriteBasic.All, PrintJob.ReadWrite.All |
+|Delegated (work or school account)| PrintJob.ReadWriteBasic, PrintJob.ReadWrite, PrintJob.ReadWriteBasic.All, PrintJob.ReadWrite.All, PrinterJob.Create |
```
Comment (nilakhan) [inline api-reference/beta/api/printer-post-jobs.md:25]: Lets keep the same order which it was before

File: api-reference/beta/api/printjob-start.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ In addition to the following permissions, the user or app's tenant must have an
 
 |Permission type | Permissions (from least to most privileged) |
 |:---------------|:--------------------------------------------|
-|Delegated (work or school account)| PrintJob.Create, PrintJob.ReadWriteBasic, PrintJob.ReadWrite, PrintJob.ReadWriteBasic.All, PrintJob.ReadWrite.All |
+|Delegated (work or school account)| PrintJob.ReadWriteBasic, PrintJob.ReadWrite, PrintJob.ReadWriteBasic.All, PrintJob.ReadWrite.All, PrinterJob.Create |
```
Comment (nilakhan) [inline api-reference/beta/api/printjob-start.md:25]: lets keep the same order here as well

File: concepts/changelog.md [hunk 5]
```diff
@@ -29,7 +29,7 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 | **Change type** | **Version**   | **Description**                          |
 |:---|:---|:---|
-| Addition | beta | Added new [Delegated permissions](/graph/permissions-reference#universal-print-permissions): <br/><ul><li>PrintJob.Create</li><li>PrinterShare.ReadBasic.All</li></ul> |
+| Addition | beta | Added new [Delegated permissions](/graph/permissions-reference#universal-print-permissions): <br/><ul><li>PrintJob.Create</li><li>PrinterShare.ReadBasic.All</li><li>PrintConnector.Read.All</li><li>PrintConnector.ReadWrite.All</li><li>PrintSettings.Read.All</li><li>PrintSettings.ReadWrite.All</li></ul> |
```
Comment (nilakhan) [inline concepts/changelog.md:32]: We also need to add Reports.Read.All scope for all reporting endpoints

File: concepts/permissions-reference.md [hunk 6]
```diff
@@ -2084,6 +2084,10 @@ Threat assessment permissions are valid only on work or school accounts.
 | _PrintJob.ReadWrite.All_ | Read and write print jobs | Allows the application to read and update the metadata and document content of print jobs on behalf of the signed-in user. | Yes | No |
 | _PrintJob.ReadWriteBasic_ | Read and write basic information of user's print jobs | Allows the application to read and update the metadata of print jobs that the signed-in user created. Does not allow access to print job document content. | No | No |
 | _PrintJob.ReadWriteBasic.All_ | Read and write basic information of print jobs | Allows the application to read and update the metadata of print jobs on behalf of the signed-in user. Does not allow access to print job document content. | Yes | No |
+| _PrintConnector.Read.All_ | Read connectors | Allows the application to read connectors on behalf of the signed-in user. | Yes | No |
+| _PrintConnector.ReadWrite.All_ | Read, update and unregister connectors | Allows the application to read, update and delete (unregister) connectors on behalf of the signed-in user. | Yes | No |
```
Comment (nilakhan) [inline concepts/permissions-reference.md:2088]: Read, update and unregister connectors => Read and write print connectors. 
Description =>  
Allows the application to read and write print connectors on behalf of the signed-in user.

image

Comment (nilakhan) [reply concepts/permissions-reference.md:2088]: We should use strings that have already been published in Azure Portal (we can see them while trying to configure permissions in portal).

**Review by nilakhan — CHANGES_REQUESTED** [review]

File: concepts/changelog.md [hunk 7]
```diff
@@ -29,7 +29,7 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 | **Change type** | **Version**   | **Description**                          |
 |:---|:---|:---|
-| Addition | beta | Added new [Delegated permissions](/graph/permissions-reference#universal-print-permissions): <br/><ul><li>PrintJob.Create</li><li>PrinterShare.ReadBasic.All</li></ul> |
+| Addition | beta | Added new [Delegated permissions](/graph/permissions-reference#universal-print-permissions): <br/><ul><li>PrintJob.Create</li><li>PrinterShare.ReadBasic.All</li><li>PrintConnector.Read.All</li><li>PrintConnector.ReadWrite.All</li><li>PrintSettings.Read.All</li><li>PrintSettings.ReadWrite.All</li><li>Reports.Read.All</li></ul> |
```
Comment (Lauragra) [inline concepts/changelog.md:32]: Please update the entry in the changelog JSON file for this change as well.

File: changelog/Microsoft.PrintService.json [hunk 8]
```diff
@@ -5,10 +5,10 @@
         {
           "Id": "0ec62390-2472-42a1-9298-2f36275c58bf",
           "ApiChange": "Resource",
-          "ChangedApiName": "printJob, printerShare",
+          "ChangedApiName": "printJob, printerShare, connector, printSetting",
```
Comment (nilakhan) [inline changelog/Microsoft.PrintService.json:8]: printConnector, printSettings

File: concepts/permissions-reference.md [hunk 9]
```diff
@@ -2084,6 +2084,11 @@ Threat assessment permissions are valid only on work or school accounts.
...
 | _PrintJob.ReadWriteBasic.All_ | Read and write basic information of print jobs | Allows the application to read and update the metadata of print jobs on behalf of the signed-in user. Does not allow access to print job document content. | Yes | No |
+| _PrintConnector.Read.All_ | Read connectors | Allows the application to read connectors on behalf of the signed-in user. | Yes | No |
+| _PrintConnector.ReadWrite.All_ | Read and write print connectors | Allows the application to read and write print connectors on behalf of the signed-in user. | Yes | No |
+| _PrintSettings.Read.All_ | Read tenant-wide print settings | Allows the application to read print settings on behalf of the signed-in user. | Yes | No |
+| _PrintSettings.ReadWrite.All_ | Read and write tenant-wide print settings | Allows the application to read and update print settings on behalf of the signed-in user. | Yes | No |
+| _Reports.Read.All_ | Read all usage reports | Allows the application to read reports on behalf of the signed-in user. | Yes | No |
```
Comment (nilakhan) [inline concepts/permissions-reference.md:2091]: Reports.Read.All is not a print specific scope. Do we need to add it here as well @braedenp-msft ?

File: concepts/changelog.md (hunk 5 above)
Comment (nilakhan) [reply concepts/changelog.md:32]: We need to add this to reporting APIs as well.

**Review by nilakhan — CHANGES_REQUESTED** [review]

File: concepts/permissions-reference.md (hunk 9 above)
Comment (jahsu-MSFT) [reply concepts/permissions-reference.md]: Thanks @Lauragra for removing the Reports.Read.All and merging the PR.

**Pull request merged**